modules/dynamic-provisioning-available-plugins.adoc

```diff
@@ -1,6 +1,7 @@
 // Module included in the following assemblies
 //
 // * storage/dynamic-provisioning.adoc
+// * post_installation_configuration/storage-configuration.adoc
 
 [id="available-plug-ins_{context}"]
 = Available dynamic provisioning plug-ins
```

Comment:
For the below table, I'm confused a little. Do we want to include the internal provisioner here?  manila is a csi driver, aws efs is an extenal provisioner. If we want to include the csi driver too, we support aws ebs csi driver in OCP 4.5. It's provisioner name is `ebs.csi.aws.com`.